content/sv/toc.md

```diff
@@ -0,0 +1,38 @@
+De tolv faktorerna
+==================
+
+## [I. Codebase (Kodbas)](./codebase)
+### En kodbas med versionshantering, flera produktionssättningar
+
+## [II. Dependencies (Beroenden)](./dependencies)
+### Deklarera och isolera beroenden explicit
+
+## [III. Config (Konfiguration)](./config)
+### Lagra konfiguration i miljön
+
+## [IV. Backing services (Stödtjänster)](./backing-services)
+### Behandla stödstjänster som anslutna resurser
+
+## [V. Build, release, run (Bygga, produktionssätta, köra)](./build-release-run)
+### Strikt separation mellan bygg- och körstadie
+
+## [VI. Processes (Arbetsprocesser)](./processes)
+### Exekvera applikationen som en eller flera tillståndslösa processer
+
+## [VII. Port binding (Portbindning)](./port-binding)
+### Göra tjänster tillgängliga genom portbindning
+
+## [VIII. Concurrency (Samtidighet)](./concurrency)
+### Skala ut systemet inom processmodellen
+
+## [IX. Disposability (Engångsbruk)](./disposability)
```

Comment:
```suggestion
## [IX. Disposability (Till förfogande)](./disposability)
```

I wonder if ["Till förfogande"](https://sv.bab.la/lexikon/svensk-engelsk/till-f%C3%B6rfogande) is better here, as Engångsbruk translates to "One-time-use" and is not completely in line with how I view disposability. I could be wrong here though.

Reply:
I don't think "till förfogande" is quite the right translation in this context but how do you feel about "Tillgänglighet"? It would also align well with "Samtidighet" above.
Another possible applicable translation would be "Tillgängliggörande".

Reply:
Both those work, prefer "Tillgängliggörande" due to "Tillgänglihet" tying so close to availability which I connect to uptime.

Reply:
In each separate file we use the Swedish word first, and the English word in parentheses. Should we have that in the TOC too? What do you think?

Reply:
I'll make sure that all the subtitles are aligned with the subtitles in each file.